packages/plugin-compiler/src/test/resources/schema/input/Schema.kt

```diff
@@ -23,7 +23,7 @@ class A : RealmObject
 class B : RealmObject
 class C : RealmObject
 
-val conf1 = RealmConfiguration.Builder()
+val conf1 = RealmConfiguration.Builder(setOf())
     .schema(A::class, B::class, C::class)
```

Comment:
I wonder why we should keep `schema` if we already set it via the builder argument. One could also set some class on the builder and some on the `schema` method which could lead to confusion (more error-prone). Can't we just remove the `schema` since it's redundant?